sdf.md

```diff
@@ -515,6 +612,9 @@ Modular composition of definitions enables an existing definition (could be in t
 ### Use of the "odmRef" keyword to re-use a definition
 An existing definition may be used as a template for a new definition, that is, a new definition is created in the target namespace which uses the defined qualities of some existing definition. This pattern will use the keyword "odmRef" as a quality of a new definition with a value consisting of a reference to the existing definition that is to be used as a template. Optionally, new qualities may be added and values of optional qualities and quality values may be defined.
 
+ISSUE: Can qualities from the source definition be overridden?
+The above only says "added".
+
```

Comment:
We should discuss this to see if there are issues with redefining the values of qualities when using odmRef to create a definition. Maybe we should consider a keyword like "default" to indicate that a quality may be redefined
```
"odmData": 
  "foo" : {
    "type": "number",
    "const": 5,
    "description": "Footacular variable can never be changed"
  }
  "bar" : {
    "type": "number",
    "default": 5,
    "description": "barmatic value may be set in the definition that uses odmRef and redefines the value of bar"
  }